vhdbuilder/packer/test/run-test.sh

```diff
@@ -11,8 +11,8 @@ TEST_VM_ADMIN_PASSWORD="TestVM@$(date +%s)"
 set -x
 
 if [ "$OS_TYPE" == "Linux" ]; then
-  if [ "$IMG_SKU" == "20_04-lts-cvm" ]; then
-    echo "Skipping tests for CVM 20.04"
+  if [ "$IMG_SKU" == "20_04-lts-cvm" ] || [ "$IMG_SKU" == "cbl-mariner-1" ]; then
```

Comment:
don't we have v1 gen2 now as well? or is that covered by this?

Reply:
Good call out, I have sent up an update and will validate against the pipeline before merging